sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/repository/support/IndexPolicyCompareService.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos.repository.support;
+
+import com.azure.cosmos.models.ExcludedPath;
+import com.azure.cosmos.models.IncludedPath;
+import com.azure.cosmos.models.IndexingPolicy;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Class for determining if the index policy currenlt applied to the container matches the index policy that is
+ * specified on an entities' @CosmosIndexingPolicy annotation
+ */
+public class IndexPolicyCompareService {
+
+    public static boolean policyNeedsUpdate(IndexingPolicy existingPolicy, IndexingPolicy newPolicy) {
+        return new IndexPolicyCompareService().needsUpdate(existingPolicy, newPolicy);
```

Comment:
@Blackbaud-EricSlater - why do we need an instance here ?
Can't we static methods for this service ?
Since this is a stateless service, creating instance here is redundant in my opinion, thoughts ?

Reply:
@Blackbaud-EricSlater - can you please implement the above changes so we can get this PR in ?
I am currently out on vacation, and won't be able to review after tomorrow. 
In case you are out too, we can merge it in January.

Reply:
fixed